test_project/lib/extends/getter.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:autoequal/autoequal.dart';
+import 'package:equatable/equatable.dart';
+
+part 'getter.g.dart';
+
+@autoequal
+class Getter extends Equatable {
+  const Getter();
+
+  set one(String newValue) => print(newValue);
```

Comment:
It seems that it should be getter instead of a setter

```suggestion
  String get one => 'one';
```